README.md

```diff
@@ -22,17 +22,15 @@ The following files constitute the KEVM semantics:
 
 -   [krypto.md](krypto.md) sets up some basic cryptographic primitives.
 -   [network.md](network.md) provides the status codes which are reported to an Ethereum client on execution exceptions.
--   [json-rpc.md](json-rpc.md) is an implementation of JSON RPC in K.
```

Comment:
This change should be reverted, since this file will continue to live here for now.